lte/gateway/c/core/BUILD.bazel

```diff
@@ -1070,6 +1070,7 @@ MME_DEPS = [
     "@system_libraries//:libfd",
     "@system_libraries//:libgnutls",
     "@system_libraries//:libnettle",
+    "@system_libraries//:libsqlite3-dev",
```

Comment:
The test is failing because this library does not exists. This change also must be done in #13470.